apis/v1alpha1/gatewayclass_types.go

```diff
@@ -115,6 +123,10 @@ type GatewayClassStatus struct {
 	// Conditions is the current status from the controller for
 	// this GatewayClass.
 	//
+	// Controllers should prefer to publish Conditions using values of
+	// [GatewayClassConditionType](#networking.x-k8s.io/v1alpha1.GatewayClassConditionType")
```

Comment:
Same comment as above - I'd personally prefer to avoid manually adding markdown links manually here.